lms/templates/help_modal.html

```diff
@@ -15,7 +15,7 @@
 % if settings.FEATURES.get('ENABLE_FEEDBACK_SUBMISSION', False):
 
 <div class="help-tab">
-  <a href="#help-modal" rel="leanModal" role="button">${_("Help")}</a>
+  <a href="#help-modal" rel="leanModal" role="button">${_("Feedback")}</a>
```

Comment:
@clrux, the link is going somewhere. It opens a different web page (the RTD doc) in a different pane. I got UX advice that we should indicate that the link opens in a new tab/window with an icon. I have that scheduled as a follow on story. I thought opening a web page would be a perfect fit for an <a href="..." />, no?